client/src/tutorials/SplTokenVault/files/lib.rs

```diff
@@ -0,0 +1,86 @@
+use anchor_lang::prelude::*;
+use anchor_spl::token::{Mint, Token, TokenAccount, Transfer};
```

Comment:
These warnings come from the code that will be pasted in during the tutorial. Maybe it would be better to have all the code in the files already? It maybe confusing to people if the have to add imports and change _ctx to ctx etc. 
So either leave the warning or already put the whole code it. I tend to having all the code in because it makes it easier to work with and an easy reference when you want to look things up.

Reply:
I removed the warning and added to the tutorial text to add transfer to the imports.

Reply:
Ah sorry, I thought it was just a forgotten import.